src/current/cockroachcloud/metrics-row-level-ttl.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Row-Level TTL metrics
+summary: The Row-Level TTL metrics let you monitor the performance of your Row-Level TTL jobs.
+toc: true
+---
+
+{{site.data.alerts.callout_info}}
+These graphs are available for CockroachDB {{ site.data.products.standard }} deployments. For graphs available to CockroachDB {{ site.data.products.advanced }} or {{ site.data.products.basic }} deployments, refer to the [CockroachDB {{ site.data.products.cloud }} Console Metrics page]({% link cockroachcloud/metrics.md %}#cockroachdb-cloud-console-metrics-page).
+{{site.data.alerts.end}}
+
+The Row-Level TTL metrics let you monitor the performance of your [Row-Level TTL jobs]({% link {{site.current_cloud_version}}/row-level-ttl.md %}).
+
+To view these graphs, select a cluster from the [**Clusters** page]({% link cockroachcloud/cluster-management.md %}#view-clusters-page), and click **Metrics** in the **Monitoring** section of the left side navigation. On the **Metrics** page, click the **Row-Level TTL** tab.
+
+## Time interval selection
+
+The time interval selector at the top of each tab allows you to filter the view for a predefined or custom time interval. Use the navigation buttons to move to the previous, next, or current time interval. When you select a time interval, the same interval is selected for all charts on the **Metrics** page.
+
+{% assign tab = "Row-Level TTL" %}
+{% assign graphs = site.data.metrics | where_exp: "graphs", "graphs.metric_ui_tab contains tab" | map: "metric_ui_graph" | uniq %}
+
+{% for g in graphs %} {% comment %} Iterate through the graphs. {% endcomment %}
+
+## {{ g }}
+
+{% assign metrics = site.data.metrics | where: "metric_ui_graph", g %}
+{% comment %} Fetch all metrics for given metric_ui_tab. {% endcomment %}
+
+<table>
```

Comment:
are all of these HTML tables the same?  it seems like the difference is above where we do `assign tab = "Row-Level TTL"` and `assign graphs = ...`, right?

if yes, idk if we want to do this in this PR, but it feels like the HTML piece could be an include if that doesn't change across any of these pages

that said this PR is already HUUUUUUUUGE so maybe we just get it in and refactor later